all/embedded/pom.xml

```diff
@@ -138,6 +138,23 @@
          <artifactId>c3p0</artifactId>
          <optional>${uberjar.deps.optional}</optional>
       </dependency>
+
+      <dependency>
+         <groupId>com.zaxxer</groupId>
+         <artifactId>HikariCP</artifactId>
+         <optional>${uberjar.deps.optional}</optional>
+      </dependency>
+      <dependency>
+         <groupId>org.jboss.slf4j</groupId>
+         <artifactId>slf4j-jboss-logging</artifactId>
+         <optional>${uberjar.deps.optional}</optional>
+      </dependency>
+      <dependency>
+         <groupId>org.slf4j</groupId>
```

Comment:
Is it possible to relocate all those dependencies? It is very likely that someone will have SLF4J on his classpath and there might be a version clash.